tutorial/write-custom-widget.md

```diff
@@ -69,9 +69,9 @@ type WidgetRenderer interface {
 As you can see the `Layout(Size)` and `MinSize()` functions are similar to the
 `fyne.Layout` interface, without the `[]fyne.CanvasObject` parameter - this is because a widget does need to be laid out but it controls which objects will be included.
 
-The `Refresh()` method is triggered when the widget this renderer draws has changed or if the theme is altered.
```

Comment:
Why the change from method to function? These refer to calls on an object instance which is typically referred to as method to differentiate from the functions that are not attached to a type.

Reply:
I will revert to method. This is a throwback to java 'methods', in go the keyword is 'func'. 
Is it correct to say that static functions (e.g. constructors) are 'functions' and that functions bound to a struct are 'methods'?

Just curious!

Reply:
Yes that is correct. Anything that is a value or pointer receiver (basically anything that you call with your created type.something()) are methods. Everything else is functions.

Reply:
I have not abandoned this. I am on holiday. I will return to this as soon as I am able.

Reply:
I have made changes but I am not sure what happens next. Do I resolve the conversations as I  or the person who started the conversation. 

I have also been pushing multiple commits. I do this because I use two different laptops to make the changes and I need to sync between them. One in my home office and one I take with me. I am not sure if this is the correct approach.

The changes are considerable. I assume another review will be done and a new set of conversations will be generated. How is this process managed. The corporate environment was somewhat different.

Regards
Stuart

Reply:
We prefer small commits where each has separate area/intent.
And the person raising the question usually resolves it so they can confirm they are happy with the resolution.
You will find comments appear here if anything more needs changed - GitHub notifies collaborators when new commits are pushed.